plugins/inputs/intel_powerstat/README.md

```diff
@@ -73,268 +110,279 @@ no per-CPU metrics are collected:
 [[inputs.intel_powerstat]]
 ```
 
-## Example: Configuration for CPU Temperature and CPU Frequency
+### Example: Configuration for CPU Temperature and CPU Frequency
 
 This configuration allows getting default processor package specific metrics,
-plus subset of per-CPU metrics (CPU Temperature and CPU Frequency):
+plus subset of per-CPU metrics (CPU Temperature and CPU Frequency) which will be
+gathered only for `cpu_id = 0`:
 
 ```toml
 [[inputs.intel_powerstat]]
   cpu_metrics = ["cpu_frequency", "cpu_temperature"]
+  included_cpus = ["0"]
 ```
 
-## Example: Configuration for CPU Temperature and CPU Frequency without default package metrics
+### Example: Configuration for CPU Temperature and CPU Frequency without default package metrics
 
-This configuration allows getting only a subset of per-CPU metrics (CPU
-Temperature and CPU Frequency):
+This configuration allows getting only a subset of per-CPU metrics
+(CPU Temperature and CPU Frequency) which will be gathered for
+all `cpus` except `cpu_id = ["1-3"]`:
 
 ```toml
 [[inputs.intel_powerstat]]
   package_metrics = []
   cpu_metrics = ["cpu_frequency", "cpu_temperature"]
+  excluded_cpus = ["1-3"]
 ```
 
-## Example: Configuration with all available metrics
+### Example: Configuration with all available metrics
 
 This configuration allows getting all processor package specific metrics and
 all per-CPU metrics:
 
 ```toml
 [[inputs.intel_powerstat]]
-  package_metrics = ["current_power_consumption", "current_dram_power_consumption", "thermal_design_power", "max_turbo_frequency", "uncore_frequency"]
-  cpu_metrics = ["cpu_frequency", "cpu_busy_frequency", "cpu_temperature", "cpu_c0_state_residency", "cpu_c1_state_residency", "cpu_c6_state_residency"]
+  package_metrics = ["current_power_consumption", "current_dram_power_consumption", "thermal_design_power", "max_turbo_frequency", "uncore_frequency", "cpu_base_frequency"]
+  cpu_metrics = ["cpu_frequency", "cpu_c0_state_residency", "cpu_c1_state_residency", "cpu_c3_state_residency", "cpu_c6_state_residency", "cpu_c7_state_residency", "cpu_temperature", "cpu_busy_frequency", "cpu_c0_substate_c01", "cpu_c0_substate_c02", "cpu_c0_substate_c0_wait"]
```

Comment:
Don't provide examples with deprecated options (`cpu_busy_cycles`)

Reply:
Done as part of commit [e2de6f6b](https://github.com/influxdata/telegraf/pull/14363/commits/e2de6f6b2577fe4e9ec7b420132b2750fe7e2654).